src/NuGet.Core/NuGet.Packaging/Rules/ReferencesInNuspecMatchRefAssetsRule.cs

```diff
@@ -2,24 +2,17 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using System.Collections;
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
-using System.Reflection;
 using System.Text;
-using System.Threading.Tasks;
 using System.Xml.Linq;
-using NuGet.Client;
 using NuGet.Common;
-using NuGet.ContentModel;
 using NuGet.Frameworks;
-using NuGet.Packaging.Core;
-using NuGet.RuntimeModel;
 
 namespace NuGet.Packaging.Rules
 {
-    class ReferencesInNuspecMatchRefAssetsRule : IPackageRule
+    internal class ReferencesInNuspecMatchRefAssetsRule : IPackageRule
```

Comment:
see this [comment](https://github.com/NuGet/NuGet.Client/pull/3226#discussion_r377183570)

Reply:
[Rule #5](https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md)